exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/MetadataContext.java

```diff
@@ -41,6 +42,8 @@
 
   private PruneStatus pruneStatus = PruneStatus.NOT_STARTED;
 
+  private boolean isMetadataCacheCorrupted;
+
   public MetadataContext() {
```

Comment:
1. We can remove this constructor, it exists by default or init `pruneStatus` and `dirModifCheckMap` in it.
2. Please move enum `PruneStatus` in the end of the class